src/transforms/throttle.rs

```diff
@@ -40,6 +56,17 @@ pub struct ThrottleConfig {
 
     /// A logical condition used to exclude events from sampling.
     exclude: Option<AnyCondition>,
+
+    /// The throttling unit to use.
+    #[serde(default = "default_unit")]
+    unit: ThrottleUnit,
```

Comment:
This is how I originally named it, but I decided against it because I can see "mode" becoming overloaded. There is another ticket about throttling having different modes for when capacities are hit (dropping data vs. applying backpressure). I moved to "unit" because it was a bit more precise in terms of what the `threshold` ends up being a measure of.

I'm totally okay with making the change to `unit` if you still prefer that, but I wanted to offer the counterargument.